aws/resource_aws_elasticsearch_domain_test.go

```diff
@@ -1048,6 +1048,10 @@ resource "aws_elasticsearch_domain" "test" {
     ebs_enabled = true
     volume_size = 10
   }
+
+  timeouts {
```

Comment:
Since the deletion of clusters is run by [sweeper](https://github.com/terraform-providers/terraform-provider-aws/blob/e50d8304e4d555f4cdd8226fb13d555e6289bbf5/docs/contributing/running-and-writing-acceptance-tests.md#acceptance-test-sweepers) during the acceptance test, I think it's enough to define this configuration here once.